scripts/Engineering/gui/engineering_diffraction/tabs/calibration/model.py

```diff
@@ -0,0 +1,243 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+from __future__ import (absolute_import, division, print_function)
+
+from os import path, makedirs
+from ntpath import basename
+
+from mantid.api import AnalysisDataService as Ads
+from mantid.kernel import logger
+from mantid.simpleapi import Load, EnggVanadiumCorrections, EnggCalibrate, DeleteWorkspace, CloneWorkspace, \
+    CreateWorkspace, AppendSpectra
+from mantidqt.plotting.functions import plot
+from Engineering.EnggUtils import write_ENGINX_GSAS_iparam_file
+
+
+class CalibrationModel(object):
+    def __init__(self):
+        self.VANADIUM_INPUT_WORKSPACE_NAME = "engggui_vanadium_ws"
+        self.CURVES_WORKSPACE_NAME = "engggui_vanadium_curves"
+        self.INTEGRATED_WORKSPACE_NAME = "engggui_vanadium_integration"
+        self.out_files_root_dir = path.join(path.expanduser("~"), "Engineering_Mantid/")
+
+    def create_new_calibration(self,
+                               vanadium_path,
+                               ceria_path,
+                               plot_output,
+                               instrument,
+                               rb_num=None):
+        """
+        Create a new calibration from a vanadium run and ceria run
+        :param vanadium_path: Path to vanadium data file.
+        :param ceria_path: Path to ceria data file
+        :param plot_output: Whether the output should be plotted.
+        :param instrument: The instrument the data relates to.
+        :param rb_num: The RB number for file creation.
+        """
+
```

Comment:
Can we avoid extraneous whitespace? it tends to lead to jarring when reading over code.